### PR TITLE
Juno and Teritori Icon - fetch.js fix

### PR DESCRIPTION
**Pull request by effofxprime**
Resolved the use of the old Juno icon and added the Teritori icon.

AURA CORS issue is something I should hear back on soon.

### CODE REVIEW

**Comment by effofxprime** [comment]
fetch.js fix:
The logic didn't seem to be working right and so I just ensured that both variables in the new lines that were added were comparing it to be less than 1.  After doing so, this resolved the issue with how uptime was being displayed for vidulum.

**Comment by effofxprime** [comment]
AURA CORS is resolved.

File: src/libs/fetch.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export default class ChainFetch {
   async getLatestBlock(config = null) {
     const conf = config || this.getSelectedConfig()
     const ver = conf.sdk_version || '0.41'
-    if (ver && compareVersions(ver, '0.45') < 1) {
+    if (ver < 1 && compareVersions(ver, '0.45') < 1) {
```
Comment (liangping) [inline src/libs/fetch.js:65]: Why is this needed?

Comment (effofxprime) [reply src/libs/fetch.js:65]: This morning I woke up to a fellow validator sharing this with me:  https://media.discordapp.net/attachments/933866223211524096/1029001951876231178/unknown.png

It wasn't getting the latest blocks properly for Vidulum chain.
Looking into it further, I saw that fetch.js had just been updated.  Maybe I've misunderstood the code and what was needing to happen.

Comment (effofxprime) [reply src/libs/fetch.js:65]: https://ping.pub/vidulum/uptime

Comment (effofxprime) [reply src/libs/fetch.js:65]: I thought it was supposed to be something more like (var && compareVersion(var, '0.45')) < 1, where both were being checked for that.

I am pretty positive I've misunderstood the implementation here.

Comment (effofxprime) [reply src/libs/fetch.js]: I reverted the changes.  Incorrect fix, I apologize.

Any chain with a COSMOS SDK version below `0.45` is currently showing all blocks as missed.

**Review by liangping — APPROVED** [review]

**Comment by liangping** [comment]
I saw you disabled Aura, but the have already enabled CORS. 
it works well, wondering if you want to enable it again.

**Comment by effofxprime** [comment]
> 

The team asked me to disable it until the beginning of November.  Sorry for that back and forth, but they did enable CORS in prep for things to be fully live for the beginning of November.

File: src/libs/fetch.js (hunk 1 above)
Comment (effofxprime) [reply src/libs/fetch.js]: @liangping 
Morning/Evening ping.

```javascript
async getLatestBlock(config = null) {
    const conf = config || this.getSelectedConfig()
    const ver = conf.sdk_version || '0.41'
    const latestBlockImp = this.get('/cosmos/base/tendermint/v1beta1/blocks/latest')
    const checkImp = JSON.parse(latestBlockImp)
    // if (ver && compareVersions(ver, '0.45') < 1) {
    if (!checkImp.message.includes('Not Implemented')) {
      return this.get('/cosmos/base/tendermint/v1beta1/blocks/latest', config).then(data => Block.create(data))
    }
    return this.get('/blocks/latest', config).then(data => Block.create(data))
  }
  ```
  Before I make another pull request, I figured it'd be best to discuss it first.
  
  The above code is non-working but I believe demonstrates what I am thinking/trying to accomplish.
  
  Instead of checking for the COSMOS SDK version, I think I would be better to check if the API call is implemented or not.
  For example: `https://api-lum.degeno.de/cosmos/base/tendermint/v1beta1/blocks/latest`  versus `https://api-lum.degeno.de/blocks/latest`
  With the recent update, it's using the firs method for uptime from what I can tell.  Since the json output of that URL is
  ```json
  {
  "code": 12,
  "message": "Not Implemented",
  "details": [
  ]
}
```
I thought it would be best to check for that instead, if it returns `Not Implemented` then use the other API call location, otherwise, continue with the other method.

What are your thoughts?  Also, if you think this would be a good way to go about it, what am I doing wrong?

Comment (effofxprime) [reply src/libs/fetch.js]: In the case of Vidulum, both API calls return information.
https://mainnet-lcd.vidulum.app/cosmos/base/tendermint/v1beta1/blocks/latest
AND
https://mainnet-lcd.vidulum.app/blocks/latest

But only when the later is used will block information be displayed and uptime be shown properly.
